projects/opendr_ws/src/perception/scripts/landmark_based_facial_expression_recognition.py

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/env python
+# Copyright 2020-2021 OpenDR European Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import rospy
+import torch
+import numpy as np
+import pandas as pd
+from std_msgs.msg import String
+from vision_msgs.msg import ObjectHypothesis
+from sensor_msgs.msg import Image as ROS_Image
+from opendr_bridge import ROSBridge
+from opendr.perception.facial_expression_recognition.landmark_based_facial_expression_recognition import \
+     ProgressiveSpatioTemporalBLNLearner
+from opendr.perception.facial_expression_recognition.landmark_based_facial_expression_recognition.\
+     algorithm.datasets.landmark_extractor import landmark_extractor
+from opendr.perception.facial_expression_recognition.landmark_based_facial_expression_recognition.\
```

Comment:
Please simplify the import statements as described in #133. If needed you can update the corresponding `__init__.py` files

Reply:
The imports are simplified now.